tests/fixtures/make_addons/expected/environments/addons/my-aurora-db.yml

```diff
@@ -27,19 +27,20 @@ Parameters:
 
 Mappings:
   myAuroraDbEnvScalingConfigurationMap:
-  
+
     development:
       DBMinCapacity: 0.5
       DBMaxCapacity: 8
-      MultiAZ:  false 
-  
+      MultiAZ: false
+
```

Comment:
Can we actually remove this extra new line?

Reply:
```suggestion
```

Reply:
I think it will be okay as line 36 is a new line as well. There are two new lines with 35 and 36.